### ISSUE TITLE
maintenance and next release

### ISSUE THREAD

**Issue by dvzrv** [notandy/ympd#189]
Hi @notandy!
I would like to be able to package ympd in the official Arch Linux repositories, but currently I'm wondering, whether ympd is still actively maintained at all.
There are many open pull requests and untriaged issues. The last release has been in 2016 with 55 commits on top of that currently.

Is there maybe something, that can be helped with? I'm sure (judging from the amount of tickets), that someone would be interested in helping maintain ympd, if you lack the time to do so. It would of course be really great to see many of the pull requests merged and a new release made!

At any rate, it would be great to get your input on that topic!

Thanks for such a great application!

**Comment by jslick** [comment]
Hi @dvzrv , I have not seen activity from the owner since the time I started watching this project.  However, it looks like it was maintained at https://github.com/SuperBFG7/ympd , which itself was forked and renamed at https://github.com/jcorporation/myMPD

**Comment by dvzrv** [comment]
@jslick yes, I stumbled upon mympd later on as well. Maybe that's more worthwhile packaging currently ;-)

**Comment by BrookeDot** [comment]
MyMPD only works with "newer" MPD versions. I am using the MPD built into Mopidy which is version 19.1 and not supported by current versions of MyMPD. Thus I am using this project in a docker container.
